src/viam/trajex/totg/trajectory.cpp

```diff
@@ -259,10 +264,13 @@ enum class integration_event : std::uint8_t {
     while (true) {
         auto current_segment = *cursor;
 
-        // Check 1: Interior extrema in circular segments (continuous-nondifferentiable case).
-        // Within a circular blend, a joint may reach a local extremum where f'_i(s) = 0
-        // while the segment continues curving. This creates a continuous-nondifferentiable
-        // point in the acceleration limit curve. Reference: Section VII-A case 2, equation 39.
+        // ========================================================================
```

Comment:
I'm not a big fan of block comments like this, with bars and CAPS. Plain english should do:
```
// Examine the current segment for switching points where the acceleration limit curve is
// continuous, but not differentiable, per VII-A Case 2 in the paper (Equation 39). These
// switching points occur on the interior of circular segments for joint extrema where f'_i(s) = 0.
```